### PR TITLE
make.ps1 default to just building daemon

### PR DESCRIPTION
**Pull request by lowenna**
Signed-off-by: John Howard <jhoward@microsoft.com>

As the docker CLI is now in a remote repo, changing the default to build just the daemon now rather than both.

### CODE REVIEW

**Comment by thaJeztah** [comment]
ping @tiborvass @johnstep

**Comment by thaJeztah** [comment]
oh! you already asked for his review 😅

File: hack/make.ps1 [hunk 1]
```diff
@@ -17,11 +17,12 @@
...
              Usage Examples (run from repo root):
-                "hack\make.ps1 -Binary" to build the binaries
-                "hack\make.ps1 -Client" to build just the client 64-bit binary
+                "hack\make.ps1 -Client" to build docker.exe client 64-bit binary (remote repo)
                 "hack\make.ps1 -TestUnit" to run unit tests
-                "hack\make.ps1 -Binary -TestUnit" to build the binaries and run unit tests
+                "hack\make.ps1 -Daemon -TestUnit" to build the daemon and run unit tests
                 "hack\make.ps1 -All" to run everything this script knows about that can run in a container
+				"hack\make.ps1" to build the daemon binary (same as -Daemon)
+				"hack\make.ps1 -Binary" shortcut to -Client and -Daemon
```
Comment (johnstep) [inline hack/make.ps1]: Replace tabs with spaces in the above two lines.

**Review by johnstep — APPROVED** [review]
LGTM with spacing nit.

File: hack/make.ps1 (hunk 1 above)
Comment (jhowardmsft) [inline hack/make.ps1]: Fixed

**Review by vdemeester — APPROVED** [review]
LGTM 🐹
@jhowardmsft question though, why not removing completely `-Client` parameter ?

**Comment by jhowardmsft** [comment]
@vdemeester ultimately yes. It would break too many workflows though to pull it quite yet.

**Comment by vdemeester** [comment]
@jhowardmsft ok, fair enough 👍

**Pull request merged**